pubspec.yaml

```diff
@@ -9,10 +9,10 @@ dependencies:
   meta: ^1.1.6
 dev_dependencies:
   args: ">=1.5.1 <3.0.0"
-  build_runner: ">=1.6.5 <3.0.0"
-  build_test: ">=0.10.8 <3.0.0"
-  build_web_compilers: ">=2.12.0 <4.0.0"
-  dependency_validator: ">=2.0.0 <4.0.0"
-  matcher: ^0.12.5
+  build_runner: ^2.1.2
+  build_test: ^2.1.3
+  build_web_compilers: ^3.0.0
+  dependency_validator: ^3.2.2
```

Comment:
Since this package is far enough upstream, we're able to upgrade to v3.2.2 which is the version with the fix for the NNBD issue. The rest of the dependency upgrades here are only for dev dependencies, which mean they don't impact downstream consumers.